### ISSUE TITLE
No cursor in "CryptPad testimonials survey"

### ISSUE THREAD

**Issue by jasonbrown1965** [cryptpad/cryptpad#859]
**Describe the bug**
No cursor in "CryptPad testimonials survey"

**Where did it happen?**
Occured on CryptPad.fr

**To Reproduce**
1. Go to Cryptpad.fr, sign in, go to drive
2. Click on User Profile icon
3. Scroll down to "CryptPad testimonials survey"
4. Click, enter text in survey text box

**Expected behavior**
To be able to see cursor blinking while editing text. 

**Screenshots**

**Browser (please complete the following information):**
 - OS: EasyOS (Linux Puppy derivative)
 - Browser: Firefox 
 -  variations: none, normal browser
 - Version 96.02.
 - Extensions installed (UBlock Origin, Privacy Badger, Decentraleyes, Https Everywhere]
 - Browser tweaks: On normal not strict mode, system loaded in RAM

**Additional context**
Cursor showing, and blinking, in Cpad drive, and rich text - no problem! 
Also showing and blinking in other browser tabs and EasyOs apps. 

_Not being able to see the cursor, at all, in your survey?_ 

Is potentially disconcerting for :

- new, newb users unfamiliar with the interface, wondering if they've done something wrong 
- power users who know they've done nothing wrong, but wondering if cpad is really up to the task of hosting surveys or other organisational efforts online?

In other words, a minor if potentially costly block to both new users and potential new customers.

**Comment by mthld** [comment]
Reproducing this on Firefox v103.0.1 on macOS v12.5. Also when you delete characters the cursor disappear too. See the screencast below:

https://user-images.githubusercontent.com/41020854/183423785-f2346861-c933-454d-bd3c-f5beb73796f1.mov
